generators/generator-bot-assistant-core/README.md

```diff
@@ -1,9 +1,54 @@
 # @microsoft/generator-bot-assistant-core [![NPM version][npm-image]][npm-url]
-> Yeoman generator for creating an Adaptive bot built on the Azure Bot Framework using the Assistant Core template.
 
-## Installation
+> Yeoman generator for creating an assistant-style bot with NLP built on the Azure Bot Framework.
 
-First, install [Yeoman](http://yeoman.io) and @microsoft/generator-bot-assistant-core using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)):
+The Assistant Core template creates an assistant-style conversational bot containing dialogs for the common conversational flows, with example language models and triggers routing to those dialogs. Includes support for
+
+- Greeting new and returning users
+- Asking for help
+- Cancelling a dialog
+- Submitting feedback about the bot
+- Error handling in conversations
+- Repeat the previous question
+- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
+- Disambiguation of NLP results
+
+## What this template is for
+
+Use this template if you want to...
+
+- Create an assistant-style, or advanced conversational bot
+- See examples of more complex conversational flows, and more advanced language understanding and generation
+
+## Packages
+
+Your bot can use the [Azure Bot Framework component model](https://aka.ms/ComponentTemplateDocumentation) to extend the base functionality. From Composer, use the Package Manager to discover additional packages you can add to your bot.
+
+This bot starts with the following packages:
+
+- [Help and Cancel intent handling](https://www.nuget.org/packages/Microsoft.Bot.Components.HelpAndCancel/)
+
+## Languages
+
+- English (en-us)
+
+## Azure Resource Deployment
+
+To run this bot you'll need the resources listed below. Create a publishing profile in Composer to provision and publish to your Azure resources for your bot.
+
+- [LUIS][luis], or another recognizer of your choice
+- [QnA Maker](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/overview/overview)
+- A storage solution for persistent state storage like Azure CosmosDB
+
+## Using this template
+
+### From Composer
+
+From Composer you'll use the **New** button on the **Home** screen to create a new bot. After creation, Composer will guide you through making customizations to your bot. If you'd like to extend your bot with code, you can open up your bot using your favorite IDE (like Visual Studio) from the location you choose during the creation flow.
+
+### From the command-line
+
+First, install [Yeoman](http://yeoman.io) and @microsoft/generator-bot-conversational-core using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)):
```

Comment:
Instructions here need to point to assistant-core, not conversational-core.

Reply:
fixed